addons/event_booth_sale/models/sale_order_line.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    event_booth_category_id = fields.Many2one('event.booth.category', string='Booths Category', ondelete='set null')
+    event_booth_pending_ids = fields.Many2many(
```

Comment:
Worth an explanation in the "help" attribute maybe? Took me a few minutes to understand that these are stored in the registration model until they are confirmed. And that when they are confirmed the so_id is set on the booth directly.